com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/zcl/ZclCluster.java

```diff
@@ -824,15 +824,12 @@ public void removeAttributeListener(final ZclAttributeListener listener) {
      * Notify attribute listeners of an updated {@link ZclAttribute}.
      *
      * @param attribute the {@link ZclAttribute} to notify
+     * @param value the current value of the attribute
      */
-    private void notifyAttributeListener(final ZclAttribute attribute) {
+    private void notifyAttributeListener(final ZclAttribute attribute, Object value) {
         for (final ZclAttributeListener listener : attributeListeners) {
-            NotificationService.execute(new Runnable() {
-                @Override
-                public void run() {
-                    listener.attributeUpdated(attribute);
-                }
-            });
+            NotificationService.execute(() -> listener.attributeUpdated(attribute, value));
```

Comment:
Can you change this back to use the non-streaming API please.

Reply:
Hm, these aren't streams. It is a lambda, which should be supported according to the docs. I can remove them though.